### ISSUE TITLE
-k -c -C -t interaction and/or documentation is confusing

### ISSUE THREAD

**Issue by pspacek** [DNS-OARC/dnscap#244]
First what docs say:
```
     -k cmd      After each dump file specified by -w is closed, this command will be executed in a nonblocking
                 subprocess with the file name as its one argument.  It's expected that this command will be a
                 shell script that submits the finished file to a batch processing analytics system.  Note that
                 without -k, the program will exit at the first output closure due to -c or -t.

     -t lim      By default, dnscap will close its packet dump file only when interrupted.  A time limit can be
                 specified with the -t option.  When writing to a file, the packet dump file will be closed
                 when time() % lim is zero and the first file will usually be shorter than lim seconds.  If the
                 packet dump file is standard output, then after closing this file, dnscap exits.  This option
                 is inclusive with -c.

     -c lim      By default, dnscap will close its packet dump file only when interrupted.  A dump file size,
                 measured in packets, can be specified with the -c option.  If the packet dump file is standard
                 output, then after closing this file, dnscap exits.  This option is inclusive with -t.

     -C lim      By default, dnscap will close its packet dump file only when interrupted.  A dump file size,
                 measured in bytes captured, can be specified with the -C option.  If the packet dump file is
                 standard output, then after closing this file, dnscap exits.  This option is inclusive with
                 -t.
```
(version git 4ee02d4)

Problems I perceive:
- `-k` description does not mention `-C` which is also affected by it
- `-t`, `-c`, `-C` explicitly mention exiting if output is stdout but they do not mention output to other files - but dnscap exits anyway if `-k` is not specified

I find this rather confusing/unexpected because as a user I do not want to read whole man page up-down to find about possible interactions of all options. 

I propose to rephrase description of `-t`, `-c`, `-C` to something like:
> Dnscap will exit by default when the specified limit is reached. To continue specify `-k` and a output file is different than standard output.

This + mentioning `-C` at `-k` desc is minimal change to clarify docs.

Personally I would go even further and add new option like `--rotate` (or something else...) which would cause file rotation/continuation independently on `-k`. After all why it should be needed to fork anything just to rotate file? (Currently I'm I have to fork `true`.) For compatibility `-k` could imply `--rotate`.

**Comment by jelu** [comment]
> but dnscap exits anyway if -k is not specified

Example when this happens?

**Comment by pspacek** [comment]
This exists on my system after receiving first packet:
```
sudo dnscap -i lo -w /tmp/pcap -C 10
```
Tested on 82dcde1.

**Comment by jelu** [comment]
So this is a miss in documentation, probably a cut&paste from `-t` to `-c` and `-C` which should not be there. Will fix that.

About rotating, an extended option could be added to change the behavior of `-c` and `-C` but no change in behavior will be done to `-t` or `-k`.

**Comment by jelu** [comment]
Fixed in #249
